packages/cli-app/src/app.js

```diff
@@ -3,7 +3,7 @@ import exec from './exec.js';
 
 export const app = command('app', {
   description: 'Create Percy builds for native app snapshots',
-  hidden: 'This command is still in development and may not work as expected',
+  hidden: true,
```

Comment:
* was there any test for this? 
* Have you tested it locally? - follow README on how to test it if not already( `yarn global:link`)

Reply:
- There was no test for this.
- There is test written for hidden in cli-command [test](https://github.com/percy/cli/blob/master/packages/cli-command/test/help.test.js#L280).
- I have tested locally and ran a build locally as well.
- I did `yarn build` and from `node_module/.bin/percy` i ran command to test it.

Reply:
* can you take a stab and add a test in `cli-command`?

Reply:
added test.